src/components-examples/material/tabs/tab-group-harness/tab-group-harness-example.ts

```diff
@@ -0,0 +1,12 @@
+import {Component} from '@angular/core';
+
+/**
+ * @title Testing with MatTabGroupHarness
+ */
+@Component({
+  selector: 'tab-group-harness-example',
+  templateUrl: 'tab-group-harness-example.html'
+})
+export class TabGroupHarnessExample {
+  isDisabled = false;
```

Comment:
I think you can remove this from here and the template since it isn't being used in any of the tests.